services/data-pipeline-dashboard/uwsgi.service

```diff
@@ -0,0 +1,14 @@
+[Unit]
+Description=uWSGI instance to serve data-pipeline-dashboard
+After=network.target
+
+[Service]
+User=tom
```

Comment:
Should we have a more generic user name?

Reply:
I've banged my head enough to get it this far. Not sure it is worth more effort to get a machine account.